RecoLocalTracker/SiPixelRecHits/src/PixelCPEClusterRepair.cc

```diff
@@ -504,7 +506,8 @@ void PixelCPEClusterRepair::callTempReco2D(DetParam const& theDetParam,
 
   } else {
     //--- Template Reco succeeded.
-    theClusterParam.hasFilledProb_ = true;
+    if (speed_ < 0)
+      theClusterParam.hasFilledProb_ = true;
```

Comment:
apparently 2D reco always calls prob computation, regardless of speed_ value. So, this is a change in behavior